api/scripts/legal-documents/add-new-legal-document-version.js

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+
+import { usecases } from '../../src/legal-documents/domain/usecases/index.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+export class AddNewLegalDocumentVersion extends Script {
+  constructor() {
+    super({
+      description: 'Add a new legal document version.',
+      permanent: true,
+      options: {
+        type: {
+          type: 'string',
+          describe: 'Type of document (ex: "TOS", "PDP")',
+          demandOption: true,
+        },
+        service: {
+          type: 'string',
+          describe: 'Associated service (ex: "pix-app", "pix-orga",...)',
+          demandOption: true,
+        },
+        versionAt: {
+          type: 'string',
+          describe: 'Version date of the legal document, format "YYYY-MM-DD", (ex: "2020-02-27")',
+          demandOption: true,
+        },
+      },
```

Comment:
Il faut ajouter `requiresArg: true` pour être sûr d'avoir une valeur avec l'option.
- avec `demandOption: true`, le CLI va nécessairement demander cette option.
- avec `requiresArg: true`, le CLI va nécessairement demander une valeur pour l'option.

```suggestion
      options: {
        type: {
          type: 'string',
          describe: 'Type of document (ex: "TOS", "PDP")',
          demandOption: true,
          requiresArg: true,
        },
        service: {
          type: 'string',
          describe: 'Associated service (ex: "pix-app", "pix-orga",...)',
          demandOption: true,
          requiresArg: true,
        },
        versionAt: {
          type: 'string',
          describe: 'Version date of the legal document, format "YYYY-MM-DD", (ex: "2020-02-27")',
          demandOption: true,
          requiresArg: true,
        },
      },
```